nihms-filesystem-builder/src/test/java/org/dataconservancy/nihms/builder/fs/FilesystemModelBuilderTest.java

```diff
@@ -16,6 +16,84 @@
 
 package org.dataconservancy.nihms.builder.fs;
 
+import org.dataconservancy.nihms.model.NihmsSubmission;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+
 public class FilesystemModelBuilderTest {
 
+    private NihmsSubmission submission;
+    private Properties expectedProperties = new Properties();
+    private String testPropertiesFile = "FilesystemModelBuilderTest.properties";
+    private  FilesystemModelBuilder underTest = new FilesystemModelBuilder();
+
+    @Before
+    public void setup() throws Exception{
+        URL resourceFileUrl = FilesystemModelBuilderTest.class.getClassLoader().getResource(testPropertiesFile);
+        InputStream is = FilesystemModelBuilderTest.class.getClassLoader().getResourceAsStream(testPropertiesFile);
+        expectedProperties.load(is);
+        is.close();
+
+        submission = underTest.build(resourceFileUrl.getPath());
+
+        //check that we have at least one property so that we know we have processed the file
+        Assert.assertNotNull(expectedProperties.getProperty(NihmsBuilderPropertyNames.NIHMS_SUBMISSION_ID));
```

Comment:
maybe move this assertion under `is.close();`?  If the assertion fails, then there's no reason to invoke `underTest.build(...)`?